distribution/scripts/jmeter/run-performance-tests.sh

```diff
@@ -123,16 +133,18 @@ function before_execute_test_scenario() {
     echo "Cleaning databases..."
     rds_host=$(get_ssh_hostname $rds_ssh_host_alias)
     mysql -u $db_username -h $rds_host -p$db_password < /home/ubuntu/workspace/is/clean-database.sql
+    timestamp
 }
 
 function after_execute_test_scenario() {
 
+    timestamp
     is_home="/home/ubuntu/wso2is"
     write_server_metrics $wso2is_host_alias $wso2is_host_alias
     download_file "$wso2is_host_alias" $is_home/repository/logs/wso2carbon.log "$wso2is_host_alias.log"
     download_file "$wso2is_host_alias" $is_home/repository/logs/gc.log $wso2is_host_alias"_gc.log"
     download_file "$wso2is_host_alias" $is_home/repository/logs/heap-dump.hprof "$wso2is_host_alias-heap-dump.hprof"
-    download_file "$wso2is_host_alias" $is_home/repository/logs/http_access_*.log "$wso2is_http_access_log"
+    # download_file "$wso2is_host_alias" $is_home/repository/logs/http_access_*.log "$wso2is_http_access_log"
```

Comment:
Are we not downloading the access logs permanently?

Reply:
access logs are around 200-400 MB so it better to comment them out in default and make it available when it is really needed